pkg/controller/nodedaemon/daemonsets.go

```diff
@@ -99,6 +100,7 @@ func getDiskMakerDSMutateFn(
 		}
 		// to read /proc/1/mountinfo
 		ds.Spec.Template.Spec.HostPID = true
+		ds.Spec.Template.Spec.PriorityClassName = class
```

Comment:
Should be `getLocalProvisionerDSMutateFn` updated too?

Maybe you can update only `MutateAggregatedSpec()`, it seems to be called from all other places that create a DaemonSet, but please check.

Reply:
Good catch, I missed `getLocalProvisionerDSMutateFn` 

`MutateAggregatedSpec` is called from everywhere thats true, but it does not contain `deployment` to be updated, correct me if I am wrong :)

Reply:
Please do, It's better to have a single place where the class is set.

Reply:
@jsafrane done 👍🏽